### ISSUE TITLE
EC power density is NaN by default

### ISSUE THREAD

**Issue by theo-brown** [google-deepmind/torax#467]
Just came across this while testing.
If the `gaussian_ec_power_density` terms in the ECHCD source are not set, the profile is NaN and the simulation does not run.

The cause is this line:

https://github.com/google-deepmind/torax/blob/357f53234adcf2aef8b4b317796e406bc0738221/torax/sources/electron_cyclotron_source.py#L60-L62

The correct choice would be some non-zero default for the width, to prevent it becoming a delta function.
E.g.
```
gaussian_ec_power_density_width: runtime_params_lib.TimeInterpolatedInput = 1.0
```

As the `gaussian_ec_total_power = 0` by default, the default source will still be all 0s.

My mistake for letting this slip through, I didn't test rigorously enough.

**Comment by jcitrin** [comment]
Thanks for flagging. Fixed here #468 and also increased the test coverage a bit to catch the original bug.

**Comment by theo-brown** [comment]
Very speedy! You managed to patch it in the time it took me to commute home :)
